test_scripts/Defects/7_1/1585_AlertManeuver_UNSUPPORTED_RESOURCE_response_success.lua

```diff
@@ -0,0 +1,87 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1585
+---------------------------------------------------------------------------------------------------
+-- Description: HMI responds with UNSUPPORTED_RESOURCE to Speak component of AlertManeuver
```

Comment:
The description is not corresponding to the sequence from the issue. `UNSUPPORTED_RESOURCE` resultCode should be received to Navigation.AlertManeuver not to Speak component